src/hcf-release/jobs/uaa-create-user/templates/run.erb

```diff
@@ -32,9 +43,13 @@ function retry () {
     shift 2
 
     while test ${i} -lt ${max} ; do
+	printf "Trying: %s\n" "$*"
         if "$@" ; then
+	    status ' SUCCESS'
             break
         fi
+	trouble '  FAILED'
+	status "Waiting ${delay} ..."
```

Comment:
I think we might be going the other way and being _too_ verbose here.  Printing the full command once, and use `--silent`, is probably enough?